MD/EquityTemporal/EquityPricing.rdf

```diff
@@ -324,7 +324,7 @@
 		<rdfs:subClassOf rdf:resource="&fibo-md-temx-ins;Yield"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-md-temx-ins;derivedFromOutlay"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-acc-cur;hasPrice"/>
```

Comment:
I think it is a base price.  All I did was fix a bad reference.  When this provisional ontology is reviewed (or eliminated) it can be fixed at that time under an appropriate new issue.

Reply:
hasPrice is not at all the property to fix the bad reference since it has different semantics. The closest as an interim fix, which at least won't harm anything, would seem to be &fibo-md-temx-ins;derivedFrom